### PR TITLE
Create guest list for invitation

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Use formatFromFirebase for invitation.
Create guest list component for invitation

### CODE REVIEW

File: apps/festival/festival/src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -16,10 +18,12 @@ import { Observable } from 'rxjs';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DashboardComponent {
-  searchCtrl: FormControl = new FormControl('');
-  org$ = this.orgQuery.selectActive();
+  public invitationCount$ = this.invitationQuery.selectCount(invitation => invitation.status === 'pending');
+  public notificationCount$ = this.notificationQuery.selectCount();
```
Comment (ioaNikas) [inline apps/festival/festival/src/app/dashboard/dashboard.component.ts:22]: You maybe want to count only unread notifications here.

File: apps/festival/festival/src/app/dashboard/invitation/invitation.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  styleUrls: ['./invitation.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class InvitationComponent implements OnInit {
+
+  invitations$ = this.query.selectAll();
+
+  constructor(private query: InvitationQuery) { }
+
+  ngOnInit(): void {
```
Comment (ioaNikas) [inline apps/festival/festival/src/app/dashboard/invitation/invitation.component.ts:16]: Not used

File: apps/festival/festival/src/app/marketplace/marketplace.component.ts [hunk 3]
```diff
@@ -17,31 +18,24 @@ import { InvitationService, InvitationQuery } from '@blockframes/invitation/+sta
...
 })
 export class MarketplaceComponent implements OnInit, AfterViewInit, OnDestroy {
-  private subs: Subscription[] = [];
+  private sub: Subscription;
   public user$ = this.authQuery.select('profile');
   public currentWishlist$: Observable<Wishlist>;
   public wishlistCount$: Observable<number>;
-  public invitationCount$ = this.invitationQuery.selectCount();
+  public invitationCount$ = this.invitationQuery.selectCount(invitation => invitation.status === 'pending');
+  public notificationCount$ = this.notificationQuery.selectCount();
```
Comment (ioaNikas) [inline apps/festival/festival/src/app/marketplace/marketplace.component.ts:26]: Same as dashboad, maybe only unread notifications

File: apps/festival/festival/src/app/dashboard/dashboard.component.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/dashboard/dashboard.component.ts:22]: I subscribe only to unread notifications in the guard

**Pull request merged**